extensions/google-translate/src/translate-form.tsx

```diff
@@ -58,13 +59,14 @@ export default function TranslateForm() {
       actions={
         <ActionPanel>
           <ActionPanel.Section title="Generals">
-            <Action.CopyToClipboard title="Copy Translated" content={translated?.translatedText ?? ""} />
-            <Action.CopyToClipboard title="Copy Text" content={text ?? ""} />
+            <ConfigurableCopyPasteActions defaultActionsPrefix="Translated" value={translated?.translatedText ?? ""} />
+            <Action.CopyToClipboard title="Copy Text" content={text ?? ""} />,
```

Comment:
```suggestion
            <Action.CopyToClipboard title="Copy Text" content={text ?? ""} />
```

Reply:
can you please remove this commas here and below? I missed it when moved it from array